### ISSUE TITLE
Support Typetalk?

### ISSUE THREAD

**Issue by is2ei** [valeriansaliou/vigil#33]
Hi, how about adding notifier for [Typetalk](https://www.typetalk.com/) ?
If the suggestion is acceptable, I'd love to send PR. 🚀

**Comment by valeriansaliou** [comment]
Not so sure about adding it directly to Vigil as it’s not so popular to me.

Though, to accept virtually any system in the future, what I can do is add a “Web Hook” notifier so that Vigil can call an HTTP endpoint of your make and then you push the notification to whichever system you want.

Is it okay if we do it this way?

**Comment by valeriansaliou** [comment]
Of course, existing notifiers are used enough to be kept inside the core for convenience.

**Comment by valeriansaliou** [comment]
I’m accepting PRs for a Web Hook notifier :)

**Comment by valeriansaliou** [comment]
@is2ei A Web Hook notifier has been added to Vigil. You can now build your own Typetalk notifier (eg. via a PHP script or something) upon it.

It's been added in this commit: fa09a6901cc1687acb325e3b435d8e6eb3bfc05e

In your Vigil configuration, you'd need to add `[notify.webhook]` with `hook_url` with the URL to the Web Hooks endpoint.

Upon status change (or status 'is still dead' reminder), you'll receive a JSON payload such as:

```json
{"type":"changed","status":"dead","time":"08:58:28 UTC+0200","replicas":["web:core:tcp://edge-3.pool.net.crisp.chat:80"],"page":{"title":"Crisp Status","url":"https://status.crisp.chat/"}}
```

**Comment by valeriansaliou** [comment]
Released in `v1.10.0`
